src/WebJobs.Script/Utility.cs

```diff
@@ -572,6 +572,32 @@ public static bool IsHttporManualTrigger(string triggerType)
             return false;
         }
 
+        public static bool IsSimpleHttpTriggerFunction(FunctionMetadata functionMetadata)
+        {
+            if (functionMetadata == null)
+            {
+                throw new ArgumentNullException(nameof(functionMetadata));
+            }
+            if (functionMetadata.InputBindings == null || functionMetadata.OutputBindings == null)
```

Comment:
what case is this handling?

Reply:
for completeness of this helper method. `FunctionMetdata` without InputBindings/OutputBindings is valid